source/uri-options/uri-options.rst

```diff
@@ -278,21 +284,30 @@ pertaining to URI options apply here.
      - required if the driver's language/runtime allows non-global configuration
      - Password to decrypt the client private key to be used for TLS connections
 
-   * - tlsInsecure
+   * - tlsDisableCertificateRevocationCheck
```

Comment:
Arrange options in alphabetical order

Reply:
In the future, this should probably be a separate PR; it makes it less clear what the actual changes to this spec in the PR are

Reply:
Noted!